Week1/homework/js-exercises/compareArrays.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+const arr1 =["Hello", 123, true, {name:"Dogan"}];
+
+const arr2=[99, true, "Dogan", king, null, {name:"Dogan"}, "London"];
+
+console.log(arr1.length);
+console.log(arr2.length);
+
+if (arr1.length === arr2.length) {console.log("They are the same!);}
+else {console.log("two different sizes");}
```

Comment:
If you want to write it shorthand you could do `check ? true : false` or like so `console.log(check ? true : false)`

In this case you could have done (this is a tip):

```
console.log(arr1.length === arr2.length ? "They are the same! : "They are different");
```

It's a bit less code and more readable 👍